<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>481</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1121</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar:tab{width:120}
QTabBar:tab{height:30}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>中央控制器</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>451</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>运行线路信息</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_3">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>1061</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabBar:tab{width:120}
QTabBar:tab{height:30}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_13">
        <attribute name="title">
         <string>设置车站信息</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1101</width>
           <height>581</height>
          </rect>
         </property>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>车站名称中文</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>车站名称英文</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行站代码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行站代码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行站间距</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行站间距</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行开门侧</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行开门侧</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行预报站广播触发距离</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行预报站广播触发距离</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行到站广播触发距离</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行到站广播触发距离</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上行越站</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下行越站</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_14">
        <attribute name="title">
         <string>会读车站信息</string>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>参数设定</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>设备配置</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>971</width>
         <height>441</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_15">
        <attribute name="title">
         <string>编辑</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_16">
        <attribute name="title">
         <string>重置</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>广播控制盒</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ip地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起始站代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点站代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>当前站代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>下一站代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对讲报警器编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>话筒音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>广播监听音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对讲监听音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行反向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本号</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>动态地图</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ip地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>背光亮度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设定背光</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本信息</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>数字功率放大器</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>当前播放优先级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测性能</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>LED目的地显示屏</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ip地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>模式切换</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>模式状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本信息</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>客室LED显示屏</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ip地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测性能</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本信息</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>紧急报警器</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所在列车号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ip地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>报警状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>话筒音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对讲监听音量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运行状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本信息</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>设备故障</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>131</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>车厢编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>系统配置</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>1261</width>
     <height>131</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>设备名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>设备编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>所在列车号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>设备ip地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>起始站代码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>终点站代码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>当前站代码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下一站代码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>动态地图亮度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>乘客信息显示屏滚动速度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>运行反向</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>运行状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>版本号</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>设备状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>车站编辑</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
